docs/how-to-add-a-new-repository.md

```diff
@@ -0,0 +1,41 @@
+# Adding or moving a repository to the KubeVirt organization
+
+In certain occasions it makes sense to create or move a repository to the KubeVirt GitHub organization. 
+
+However, since we don't want this to go unnoticed for several reasons, i.e. the possible advertisement that we have a new and helpful project for the KubeVirt community, we are advocating a process that every repository needs to go through before it is added to the KubeVirt GitHub org.
+
+## Questions to ask
+
+* why is the repo a good fit for the org?
+* does the licensing fit the overall licensing in KubeVirt?
+* are all maintainers ok with transitioning?
+* is CI support required and if so do we have the CI capacity left? Ask the [CI maintainers](../OWNERS_ALIASES) to make sure of this
+* are minimal parameters for repo and maintainer team defined (description, maintainers)?
+
+## Repository and team configuration
```

Comment:
Add description about activating branch protection?

Reply:
Added link to merge automation docs.